src/pip/_internal/vcs/bazaar.py

```diff
@@ -40,13 +39,11 @@ def export(self, location):
         if os.path.exists(location):
             rmtree(location)
 
-        with TempDirectory(kind="export") as temp_dir:
-            self.unpack(temp_dir.path)
-
-            self.run_command(
-                ['export', location],
-                cwd=temp_dir.path, show_stdout=False,
-            )
+        url, rev = self.get_url_rev()
```

Comment:
You should be calling ‘make_rev_options()’ here after this like the beginning of ‘obtain()’ does. ‘get_base_rev_args()’ is meant for use by the RevOptions class rather than by VersionControl directly.

Reply:
Done.